tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1518,7 +1518,7 @@ generic_config_updater/test_multiasic_linkcrc.py:
 
 generic_config_updater/test_packet_trimming_config.py:
   skip:
-    reason: "KVM do not support the feature"
+    reason: "Packet trimming cases require PR https://github.com/sonic-net/sonic-buildimage/pull/22869, but KVM does not support it, so skip trimming case on KVM"
```

Comment:
I am seeing this PR is already merged. Should we just say KVM doesn't support packet trimming config?